src/domain/transfer/index.js

```diff
@@ -134,30 +117,25 @@ const fulfil = async (headers, dataUri, payload, params, span) => {
  * @returns {boolean} Returns true on successful publishing of message to kafka, throws error on failures
  */
 const getTransferById = async (headers, params, span) => {
-  Logger.isDebugEnabled && Logger.debug(`domain::transfer::transferById::start(${params.id}, ${JSON.stringify(headers)}`)
+  logger.debug(`domain::transfer::transferById::start(${params.id}, ${JSON.stringify(headers)}`)
   try {
-    const state = StreamingProtocol.createEventState(generalEnum.Events.EventStatus.SUCCESS.status, generalEnum.Events.EventStatus.SUCCESS.code, generalEnum.Events.EventStatus.SUCCESS.description)
-    const event = StreamingProtocol.createEventMetadata(generalEnum.Events.Event.Type.GET, generalEnum.Events.Event.Type.GET, state)
+    const state = dto.eventStateDto()
+    const event = StreamingProtocol.createEventMetadata(Type.GET, Type.GET, state)
     const metadata = StreamingProtocol.createMetadata(params.id, event)
     let messageProtocol = StreamingProtocol.createMessageFromRequest(params.id, { headers, dataUri: undefined, params }, headers[generalEnum.Http.Headers.FSPIOP.DESTINATION], headers[generalEnum.Http.Headers.FSPIOP.SOURCE], metadata)
-    const topicConfig = KafkaUtil.createGeneralTopicConf(Config.KAFKA_CONFIG.TOPIC_TEMPLATES.GENERAL_TOPIC_TEMPLATE.TEMPLATE, generalEnum.Events.Event.Action.TRANSFER, generalEnum.Events.Event.Action.GET)
-    const kafkaConfig = KafkaUtil.getKafkaConfig(Config.KAFKA_CONFIG, generalEnum.Kafka.Config.PRODUCER, generalEnum.Events.Event.Action.TRANSFER.toUpperCase(), generalEnum.Events.Event.Action.GET.toUpperCase())
-    Logger.isInfoEnabled && Logger.info(`domain::transfer::get::messageProtocol - ${messageProtocol}`)
-    Logger.isInfoEnabled && Logger.info(`domain::transfer::get::topicConfig - ${topicConfig}`)
-    Logger.isInfoEnabled && Logger.info(`domain::transfer::get::kafkaConfig - ${kafkaConfig}`)
-    // TODO: re-enable once we are able to configure the log-level
-    // await span.debug({
-    //   messageProtocol,
-    //   topicName: topicConfig.topicName,
-    //   clientId: kafkaConfig.rdkafkaConf['client.id']
-    // })
+    const topicConfig = KafkaUtil.createGeneralTopicConf(Config.KAFKA_CONFIG.TOPIC_TEMPLATES.GENERAL_TOPIC_TEMPLATE.TEMPLATE, Action.TRANSFER, Action.GET)
+    const kafkaConfig = KafkaUtil.getKafkaConfig(Config.KAFKA_CONFIG, generalEnum.Kafka.Config.PRODUCER, Action.TRANSFER.toUpperCase(), Action.GET.toUpperCase())
```

Comment:
Why are we not calling the new DTO functions here?

Reply:
I didn't want to add any functional changes to the route, which is not in scope of my current task